### ISSUE TITLE
Settings for Rainbow Six Siege

### ISSUE THREAD

**Issue by MickyDNet** [fqlx/XboxKeyboardMouse#123]
I've tried countless settings, but I cant seem to get any settings working that provide a good mouse feel to the game. It either moves too quick, doesnt respond to light movements of the mouse, or has half a second of input lag (im using a local ethernet on both ways).

Has anyone found any good mouse settings that work?

**Comment by diablo2002** [comment]
Yeah Same case here I've tried plenty of configuration to this game but no one seems to work correctly , I think it's because this game have some special mouse movement,different  than other games so i stopped trying to figure out how to solve it :/ between it was the main raison that push me to use this software

**Comment by kr2001** [comment]
If your talking about input lag with controls (w,s,a,d,rmb,lmb) cant help ye but for me sensitivity is fine the settings i use are.
options in the ui,go to mouse and set values to 
mouse sensitivity (x and y) to 2000.000000000000000
Final modifier to 100.000000.
In seige set vertical and horzontal to 45 and deadzones to 5.
Should be an alright feel

**Comment by diablo2002** [comment]
I'm a pc player so i actually there is a big difference between this software and that config and the 3'rd party hardware and the pc game , so in the end of the story this software is really good in story games and games without mp beside that i don't recommend it for mp first person shooting game especially like rainbow six siege

**Comment by DavidRieman** [comment]
Did everyone try the F12/F11 calibration process?

**Comment by diablo2002** [comment]
Yes we did it works but not as good ad the original and in a game like rainbow six siege a little aim helps alot

**Comment by RenanBan** [comment]
can i give you an tip, i play R6 with the following, sense X/Y both around 650/800 im still adjusting, and the ingame sense is 100 and 80/90 to the ADS sense. works really well for me since I can turn real fast and still get a good fine aim. Of course combining with the DeadZone F11/F12 adjustment

**Comment by scxrlettz1** [comment]
Renan, Brasil ?

**Comment by RenanBan** [comment]
Sim

**Comment by T0MB5T0N3** [comment]
@DavidRieman my f12 and f11 calibration doesn't work

**Comment by 1qah** [comment]
any way too get less input delay because its really messing up my aim
